streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/Sensors.java

```diff
@@ -77,8 +82,11 @@ private static Sensor getBufferSizeOrCountSensor(final StateStore store,
         final String metricsGroup = "stream-buffer-metrics";
 
         final Map<String, String> tags = metrics.tagMap(
-            "task-id", context.taskId().toString(),
-            "buffer-id", store.name()
+            threadId,
```

Comment:
Ditto here, I think besides the threadId having other paired values in oneline is better to read / review.